packages/flutter/lib/src/material/progress_indicator.dart

```diff
@@ -854,6 +856,13 @@ class CircularProgressIndicator extends ProgressIndicator {
   )
   final bool year2023;
 
+  /// The padding around the indicator track.
+  ///
+  /// If null, then the [ProgressIndicatorThemeData.circularTrackPadding] will be
+  /// used. If that is null and [year2023] is false, then defaults to 8 pixels of
```

Comment:
Just feel 8 pixels is a little confusing. Can we say defaults to `EdgeInsets.all(4.0)`?